test/GitcoinGovernor.t.sol

```diff
@@ -427,3 +434,476 @@ contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelp
     governorAlpha.queue(_newProposalId);
   }
 }
+
+contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
+  // From GovernorCountingSimple
+  uint8 constant AGAINST = 0;
+  uint8 constant FOR = 1;
+  uint8 constant ABSTAIN = 2;
+
+  function assumeReceiver(address _receiver) public {
+    // We don't want the receiver to be the Timelock, as that would make our
+    // assertions less meaningful -- most of our tests want to confirm that
+    // proposals can cause tokens to be sent *from* the timelock to somewhere
+    // else. We also don't want the zero address as a receiver -- if our
+    // governor is sending tokens to the zero address it could just be due to a
+    // bug.
```

Comment:
Nit, it is plausible a governor might want to burn tokens for some reason
```suggestion
    // else. We also don't want the zero address as a receiver -- although
    // this is valid, it's typically not want a governor wants to do
```

Reply:
Updated comment in https://github.com/gitcoinco/Alpha-Governor-Upgrade/pull/12/commits/4216a168c97d7111604d4f0da06fb655cae7283c